src/sentry/options/defaults.py

```diff
@@ -128,6 +128,10 @@
 register("slack-v2.client-secret", flags=FLAG_PRIORITIZE_DISK)
 register("slack-v2.signing-secret", flags=FLAG_PRIORITIZE_DISK)
 
+# Slack migration
+register("slack-migration.enable-banner-check", default=False)
```

Comment:
Please set `FLAG_PRIORITIZE_DISK`. It should really be set on almost all of the options we use here. It's bad habit that people don't.

We should then follow up my setting this in getsentry for us.